src/neural/cuda/fp16_kernels.cu

```diff
@@ -248,105 +248,97 @@ void OutputInputTransformKernel_fp16_shmem_board(
   float S = 0;
   float B = 0;
 
-  if (use_bias || use_se) {
 #pragma unroll
-    for (int y = 0; y < 8; y++) {
-      half boardRow[8];
-      copyAs<uint4>(&boardRow, &BOARD(y, 0));
+  for (int y = 0; y < 8; y++) {
+    half boardRow[8];
+    copyAs<uint4>(&boardRow, &BOARD(y, 0));
 #pragma unroll
-      for (int x = 0; x < 8; x++) {
-        if (use_bias) boardRow[x] += b;
-        if (use_se) S += (float)boardRow[x];
-      }
-      if (use_bias) copyAs<uint4>(&BOARD(y, 0), &boardRow);
+    for (int x = 0; x < 8; x++) {
+      if (use_bias) boardRow[x] += b;
+      S += (float)boardRow[x];
     }
+    if (use_bias) copyAs<uint4>(&BOARD(y, 0), &boardRow);
   }
 
-  if (use_se) {
     __shared__ float shared_data[kMaxResBlockFusingSeKFp16Ampere];
-    float avg = S / 64;
```

Comment:
I think the line above is still indented when it should have shifted across with everything else.

Reply:
sorry about that. Fixed.